ecosysboard/http/coinpaprika.go

```diff
@@ -17,6 +17,8 @@
 package http
 
 import (
+	"encoding/json"
+	"github.com/kpango/glg"
```

Comment:
we should ensure that external & 3rd party packages are safe/secure to use or just add a disclaimer/warning if any are un-reviewed.

Reply:
Yes we should proceed a full review for that